lib/logstash/filters/aggregate.rb

```diff
@@ -256,6 +283,10 @@ def register
     # process lambda expression to call in each filter call
     eval("@codeblock = lambda { |event, map| #{@code} }", binding, "(aggregate filter code)")
 
+    if @timeout_code
```

Comment:
Could you just add here a comment to introduce ? 
Something like the one for @code : "process lambda expression to call in each filter call"